examples/angular/app.js

```diff
@@ -19,8 +19,4 @@ const sheet = jss.createStyleSheet(styles).attach()
 
 angular.module('myApp', []).controller('myController', $scope => {
   $scope.classes = sheet.classes
-  $scope.showSource = () => {
-    // eslint-disable-next-line no-restricted-globals
-    location.href = 'https://github.com/cssinjs/examples/tree/master/angular'
```

Comment:
I think we should keep all links to the repo (new url) because it won't be obvious how to contribute to the github repo when they see a codesandbox, we just need a different string instead of "view source",  "View the source on GitHub"

Reply:
I had to remove the "View on Github" button because it's styled by a [stylesheet](https://github.com/cssinjs/examples/blob/gh-pages/calendar/index.html#L6) outside the example files. This file would not be available to codesandbox. Can we put an absolute link to the CSS?

Reply:
Or we could leave it out and link to github same place where we are going to link to codesandbox, so that there is no need for doing it inside of example.

Reply:
Ok, when I change the documentation site repo I'll re-add the button 'View on Github'.